yeeun/pintos/threads/select_test.sh

```
@@ -0,0 +1,223 @@
+echo "Failed: ${#failed[@]}"
+for t in "${failed[@]}"; do echo "  - $t"; done
+
+# 5) 상태 파일에 PASS/FAIL 기록 (untested는 기록하지 않음)
+for t in "${passed[@]}"; do
+  status_map["$t"]="PASS"
+done
+for t in "${failed[@]}"; do
+  status_map["$t"]="FAIL"
+done
+
+> "$STATE_FILE"
```

Shellcheck (0.10.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)